ArduPlane/Plane.h

```diff
@@ -252,6 +253,8 @@ class Plane : public AP_HAL::HAL::Callbacks {
 
     // GCS selection
     AP_SerialManager serial_manager;
+    const uint8_t num_gcs = MAVLINK_COMM_NUM_BUFFERS;
```

Comment:
You are right, it's unnecessary, removed it in PR#5737. The original version of this code was based on ArduPlane 3.2, and when integrating it with 3.8 I unintentionally left in a few things in the code that made sense in the original version but are superfluous for the current one.